airflow/jobs/scheduler_job.py

```diff
@@ -1557,6 +1557,23 @@ def _schedule_dag_run(self, dag_run: DagRun, session: Session) -> int:
 
         return count
 
+    @provide_session
+    def _verify_integrity_if_dag_changed(self, dag_run: DagRun, session=None):
+        """Only run DagRun.verify integrity if Serialized DAG has changed since it is slow"""
+        latest_version = SerializedDagModel.get_latest_version_hash(dag_run.dag_id, session=session)
+        if dag_run.dag_version == latest_version:
+            self.log.debug("DAG Version not changed, skipping dagrun.verify_integrity")
```

Comment:
```suggestion
            self.log.debug("DAG %s not changed structure, skipping dagrun.verify_integrity", dag_run.dag_id)
```

Reply:
fixed in https://github.com/astronomer/airflow/pull/919/commits/ce9ffe9f9c6584f6b11f10f6c8cc16ccafde4f1f